### DISCUSSION TITLE
我反馈一下

### DISCUSSION THREAD

**Discussion by MiManchi-netizen** [share121/inter-knot#2454] [其他]
刷新总是出更新小助手到1.8.0，这是我的问题吗
![Uploading 微信图片_20240816222938.png…]()

**Comment by HifumiMizuhara** [comment]
首先，图没传上来。
其次，代码的来说不应该展现1.8.0啊……
<img width="504" alt="スクリーンショット 2024-08-16 22 38 07" src="https://github.com/user-attachments/assets/5707dae1-72cd-43ba-8869-85855dac2621">

**Reply by MiManchi-netizen** [reply]
不清楚，总之必须强制清空缓存才能正常访问

**Reply by MiManchi-netizen** [reply]
![微信图片_20240816222938](https://github.com/user-attachments/assets/f8a9f3e2-5b2a-46e2-8fa3-fe4ad7e256d7)

**Reply by HifumiMizuhara** [reply]
你的右上角还是“绳网等级”，这是什么时候的遗产……（吐槽）
Cache 太活跃了吧，清空重来试试
<img width="414" alt="スクリーンショット 2024-08-16 22 52 36" src="https://github.com/user-attachments/assets/163358e9-0a5d-48cb-8ebb-a1de3199c092">
还有就是这里用的是!==，其实有点想吐槽，这样完全体现不了现在的版本是“旧的”，只能说明不一样。比如改成“本网站适配的版本为……”都会好得多
还有其实是Update 不是 Updata……（捂脸）

**Reply by MiManchi-netizen** [reply]
楼下有好兄弟分享了另一个网址，那个没问题，我暂时用那个

**Comment by BSNHD** [comment]
https://www.绳网.top  可以试试用这个网址访问，不会出现你说的情况

**Reply by MiManchi-netizen** [reply]
好，我试试
